src/taker/mod.rs

```diff
@@ -5,7 +5,7 @@
 //! protocol workflow is described in the [protocol between takers and makers](https://github.com/citadel-tech/coinswap/blob/master/docs/dev-book.md)
 
 mod api;
-mod config;
+pub mod config;
```

Comment:
Do we need it?

Reply:
> Do we need it?

No we don't, it happened while I was trying different approach.
I've discarded this change.